webpack/config.cordova.js

```diff
@@ -0,0 +1,14 @@
+var webpack = require('webpack');
```

Comment:
**var**s usage violates our style-guide,
Please use **const** where possible, and **let** when mutations is neseccary